roles/container_runtime/defaults/main.yml

```diff
@@ -157,5 +158,5 @@ l_docker_image: "{{ openshift_docker_systemcontainer_image_override | default(l_
 
 l_is_node_system_container: "{{ (openshift_use_node_system_container | default(openshift_use_system_containers | default(false)) | bool) }}"
 
-l_crio_use_new_var_sock: "{{  (l_openshift_image_tag == 'latest') or (l_openshift_image_tag | version_compare('3.9', '>='))  | bool }}"
+l_crio_use_new_var_sock: True
```

Comment:
If we're going to backport this to 3.9, I would prefer this not be changed.

Reply:
I can remove it. I did this mainly because it was failing with a version of ``v3.9`` but it could be some corner case I was hitting.